### PR TITLE
Use fakerapi.it instead of Salesforce as data source

### PR DESCRIPTION
**Pull request by severin**

### CODE REVIEW

**Review by lukecartledge — APPROVED** [review]
LGTM 👍🏻

File: README.md [hunk 1]
```diff
@@ -2,57 +2,48 @@
...
+Breaking down the task the app should:
 
-## Using the Salesforce API
+1. Synchronize all records from a remote system via the provided API into its own database
 
-Salesforce provides an API which you will use to retrieve the dealers.
+- Consider the case that records can be deleted in the remote system
 
-For an introduction to the API: click [here](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_what_is_rest_api.htm)
+2. Display all dealers from the local database in a list and on a map
```
Comment (aaeabdo) [inline README.md:13]: Dealers -> companies

**Review by aaeabdo — APPROVED** [review]
Just a small typo besides all is good 👌

File: README.md (hunk 1 above)
Comment (aaeabdo) [reply README.md:13]: Nvm, I see you mentioned that company simulates a dealer. Probably enough.

Comment (ahorincar) [reply README.md:13]: I would actually stick with either `retailer` or `dealer` throughout the whole task description. Currently it's a mix of both.

**Review by ahorincar — APPROVED** [review]
Had a small comment about terminology, but other than that it LGTM

**Pull request merged**